### DISCUSSION TITLE
error: Failed to create HTTP server: [Errno 98] Address already in use

### DISCUSSION THREAD

**Discussion by HalowTV** [streamlink/streamlink#5074] [Q&A] [answered]
streamlink "https://www.youtube.com/watch?v=9Auq9mYxFEE" best --player-external-http --player-external-http-port 996 [cli][info] streamlink is running as root! Be careful! [cli][info] Found matching plugin okru for URL https://www.youtube.com/watch?v=9Auq9mYxFEE [cli][info] Available streams: 144p (worst), 240p, 360p, 480p, 720p (best) error: Failed to create HTTP server: [Errno 98] Address already in use

Hey i run ubuntu 18: 4 server I want to use the same gate please there is someone do to use again As cleaning the gate use again or dead port thankful for everyone's help

**Comment by mkbloke** [comment, marked as answer]
Use `sudo netstat -anp | grep :996` to find the PID/process-name currently using that port.  You'll then have to stop/kill that process in order to make port 996 available for use by Streamlink.

**Reply by HalowTV** [reply]
get this how i stop or Kill mate  i am new of this system
tcp        0      0 0.0.0.0:151             0.0.0.0:*               LISTEN      31093/python3
tcp        0      0 xxx.201.167.236:34782   195.201.167.236:996     ESTABLISHED 416/ffprobe
tcp        0      0 xxx.201.xxx.xxx:996     195.201.167.236:34782   ESTABLISHED

**Reply by gravyboat** [reply]
We can't help you learn the basics of Linux. Google it.
